packages/firestore/test/unit/specs/write_spec.test.ts

```diff
@@ -1245,4 +1245,30 @@ describeSpec('Writes:', [], () => {
       );
     }
   );
+
+  specTest(
+    'Mutation are not sent twice after primary failover',
```

Comment:
Wow, did you really write a new test to test the merge of this?  Or where did this come from?

Reply:
Yeah, I wanted to verify that we clear the write pipeline during failover.